flixel/animation/FlxAnimation.hx

```diff
@@ -204,24 +204,19 @@ class FlxAnimation extends FlxBaseAnimation
 		while (_frameTimer > delay && !finished)
 		{
 			_frameTimer -= delay;
-			
-			if (looped)
+			if (reversed)
```

Comment:
This actually makes the code _a lot_ more readable. :+1: 

Btw, previously it was hard to tell if the changes in this section were from cleanup or related to the new feature, which was what caused my confusion for the most part.